app.js

```diff
@@ -4,10 +4,16 @@ import express from 'express'
 import bodyParser from 'body-parser';
 
 let app = express();
-app.use(bodyParser.json());
- 
+
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Methods", "POST, GET, DELETE,  PUT");
```

Comment:
I remember I had an issue with `OPTIONS` method. Just if you have an issue with it,
  ```
if ('OPTIONS' === req.method) {
    res.send(200);
  }
```
will save you.

Reply:
nice